handler/handler_test.go

```diff
@@ -148,34 +154,40 @@ func Test_GetRecordHandler(t *testing.T) {
 		t.Errorf("error in GetAll request")
 	}
 
-	result := repo.Record{}
+	result := models.Record{}
 	dec := json.NewDecoder(res.Body)
 	err = dec.Decode(&result)
 	if err != nil {
 		t.Errorf("decoding out array error")
 	}
-	ExpectedResult := repo.Record{
+	ExpectedResult := models.Record{
 		ID:          "1111",
 		Type:        "reverse",
 		CaesarShift: 0,
 		Result:      "54321",
-		CreatedAt:   time.Now().Unix(),
+		CreatedAt:   testTime,
 	}
-	assert.Equal(t, ExpectedResult, result)
+
+	assert.Equal(t, ExpectedResult.ID, result.ID)
+	assert.Equal(t, ExpectedResult.Type, result.Type)
+	assert.Equal(t, ExpectedResult.CaesarShift, result.CaesarShift)
+	assert.Equal(t, ExpectedResult.Result, result.Result)
 }
 
-var UpdateRecordTestTable = []TransformRequest{
-	TransformRequest{Type: "caesar", CaesarShift: -3, Input: "abc"},
-	TransformRequest{Type: "reverse", CaesarShift: 0, Input: "54321"},
-	TransformRequest{Type: "base64", CaesarShift: 0, Input: "Man"},
+var UpdateRecordTestTable = []models.TransformRequest{
+	models.TransformRequest{Type: "caesar", CaesarShift: -3, Input: "abc"},
```

Comment:
you can skip explicit type definition when initializing slice: `models.TransformRequest{Type: "caesar", CaesarShift: -3, Input: "abc"}` -> `{Type: "caesar", CaesarShift: -3, Input: "abc"}`